src/main/java/com/launchacademy/reviews/controllers/ReviewRestController.java

```diff
@@ -0,0 +1,32 @@
+package com.launchacademy.reviews.controllers;
+
+import com.launchacademy.reviews.models.Review;
+import com.launchacademy.reviews.services.ReviewService;
+import java.util.Map;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/api/v1/reviews")
+public class ReviewRestController {
+  private ReviewService reviewService;
+
+  @Autowired
+  public ReviewRestController(ReviewService reviewService) {
+    this.reviewService = reviewService;
+  }
+
+  @PutMapping("/{id}")
+  public ResponseEntity updateVoteCount (@PathVariable Integer id, @RequestBody Map<String, String> reviewObject){
+    Review review = reviewService.findById(id).get();
+    review.setVoteCount(Integer.parseInt(reviewObject.get("finalCount")));

```

Comment:
Since a single upvote/downvote should only be changing the reviews by an increment of one, We can just add or subtract one from the current `voteCount` and then use that value in `setVoteCount` that way we don't rely on any integer coming in from the frontend.

Just something to keep in mind as that would limit this to only ever adding or subtracting one, whereas now someone could open up their devtools, change the state, and this endpoint will always persist the value sent.